custom_components/stiebel_eltron_isg/translations/en.json

```diff
@@ -24,12 +24,12 @@
                 "state_attributes": {
                     "preset_mode": {
                         "state": {
-                            "water_heating": "Water heating",
-                            "emergency": "Emergency",
-                            "program": "Program",
-                            "ready": "Ready",
-                            "auto": "Auto",
-                            "manual": "Manual"
+                            "water_heating": "Water heating operation",
```

Comment:
I propose to leave the word "operation" away in this list since it is redundant to the tile (Operation Mode) and it just eats up additional space. Please remove the word from all 6 entries in this list.